docker/package/tezos_voting_wizard.py

```diff
@@ -22,6 +22,21 @@
     "pass": "Submit a vote not influencing the result but contributing to quorum",
 }
 
+public_nodes = {
+    "mainnet": [
+        ("https://rpc.tzbeta.net", "Tezos Foundation"),
+        ("https://mainnet.api.tez.ie", "ECAD Labs"),
+        ("https://mainnet.smartpy.io", "SmartPy"),
+        ("https://teznode.letzbake.com", "LetzBake!"),
+        ("https://mainnet-tezos.giganode.io", "GigaNode"),
+    ],
+    "ithacanet": [
```

Comment:
Voting doesn't usually happen on testnet, so I don't think that we should keep a list of public nodes on `ithacanet` and `jakartanet`

Reply:
Don't we want to have an opportunity to vote on testnet? There was issue related to that #442

Reply:
Yeah, we want to be able to vote on testnets, but it's mostly for our own testing purposes. AFAIU no actual user will ever vote on testnet, and if they do, they should be able to find or run a node themself. So I agree that IMO it's better to just keep a list of mainnet nodes, as they are likely to be used.